test.properties

```diff
@@ -3448,7 +3453,7 @@
     test.batch.dist.app.servers[analytics-cloud]=tomcat
 
     test.batch.names[analytics-cloud]=\
-        functional-tomcat90-mysql57-jdk8
+        playwright-js-tomcat90-mysql57-jdk8
```

Comment:
Hey @ViniLopes20 for this change, keep this line and add back in the old line of functional-tomcat90 for both Poshi and Playwright tests

Reply:
Done, thanks @CalumR23